core/src/main/scala/dogs/BiPartition.scala

```diff
@@ -0,0 +1,36 @@
+/**
+  * Created by anicolaspp on 4/10/16.
+  */
+
+package dogs
+
+import Predef._
+import simulacrum.typeclass
+import scala.{inline,Iterable}
+import java.lang.{String,StringBuilder}
+import scala.annotation.{tailrec}
+import dogs.syntax.birds._
+import cats._
+
+
+@typeclass trait BiPartition[A] {
+  def partition(predicate: A => Boolean): (List[A], List[A])
```

Comment:
Hmm, this really seems like it should be able to be derived from some other typeclass we already have in cats. It looks a lot like separate, can we generalize separate to fit this as well? Why do we want List as the result? Can we generalize the result to a Alternative? I'm guessing the performance loss there would suck